docs/_pages/basicassertions.md

```diff
@@ -76,3 +76,15 @@ Internally, `BeBinarySerializable` uses the [Object graph comparison](objectgrap
 theObject.Should().BeBinarySerializable<MyClass>(
     options => options.Excluding(s => s.SomeNonSerializableProperty));
 ```
+
+### Don't use Equals() ###
```

Comment:
🤔 I don't think it's worth having this kind of documentation. The value of the PR is that when people accidentally use `Equals`, they get immediate and actionable feedback. Nobody is going to be searching for that in the docs.

Reply:
The reason I included this is so that if someone google searches the error (which imo is a first-class use case for an error message) they will get a hit on the docs which has the supplemental explanation.  If this is not included, they may get a hit on the indexed code (this is how I found this code and then decided to make this PR).  I agree the new error messages are much more actionable and this probably won't be needed, but it's my opinion that this is useful and it's better to land on a docs page than on a code page for non-expert developers.

Clearly, it's your project and if you still think it should be removed I will definitely do that.  Maybe I can blog about it or something.

Reply:
Fair, but then I would make it a bit more generic and just mention the `Equals is not part of Fluent Assertions. Did you mean....` part.

Reply:
I will just remove from this PR and we can discuss separately.  ty.